src/main/java/net/runelite/launcher/Launcher.java

```diff
@@ -161,13 +167,19 @@ public static void main(String[] args)
 		try
 		{
 			UIManager.setLookAndFeel(UIManager.getCrossPlatformLookAndFeelClassName());

```

Comment:
Maybe move this UIManager call inside swing thread too? Or remove it if its unnecessary